packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -375,3 +359,40 @@ export interface ITextFieldStyles {
    */
   subComponentStyles: ITextFieldSubComponentStyles;
 }
+
+/**
+ * MaskedTextField component props.
+ * {@docCategory MaskedTextField}
```

Comment:
This should stay as TextField since we don't have a separate doc page for MaskedTextField
```suggestion
 * {@docCategory TextField}
```